app/Spago.hs

```diff
@@ -278,7 +278,7 @@ bundle withMain maybeModuleName maybeTargetPath = do
         WithoutMain -> ""
 
       cmd
-        = "purs bundle './output/*/*.js'"
+        = "purs bundle output/*/*.js"
```

Comment:
Does it work if we keep the single quotes here? They are here to prevent the shell from expanding them (though I'm not sure it would happen for how we run commands)

Reply:
I'd say that's the difference, Having the shell expand them gets bundle to work on windows. I think I can also pass in quotes, but using backslashes. 

This was my punt at well, shell globbing hopefully working for both?. Alternatively it would have to be some OS detection, or maybe there is a haskell path library that can handle this.

Reply:
My point here is that `purs` supports globbing, and it's probably much faster than shell globbing (it's probably worth benchmarking the two cases tho)

We could use [`</>`](http://hackage.haskell.org/package/filepath-1.4.2.1/docs/System-FilePath-Windows.html#v:-60--47--62-) to form the path, since it supports Windows

Reply:
@kRITZCREEK suggestion works. Have rebased this commit to match.